locations/spiders/k_market_fi.py

```
@@ -0,0 +1,36 @@
+from locations.categories import apply_category, Categories
+from locations.items import Feature
+from locations.structured_data_spider import StructuredDataSpider
+
+K_CITYMARKET = {"brand": "K-Citymarket", "brand_wikidata": "Q11868561"}
+K_MARKET = {"brand": "K-Market", "brand_wikidata": "Q11868562"}
+K_SUPERMARKET = {"brand": "K-Supermarket", "brand_wikidata": "Q5408668"}
+
+
+class KMarketFISpider(SitemapSpider, StructuredDataSpider):
+    name = "k_market_fi"
+    sitemap_urls = ["https://www.k-ruoka.fi/robots.txt"]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)